### PR TITLE
Extravaganza blog post for July 2015.

### PR DESCRIPTION
**Pull request by Osmose**
@lonnen @peterbe @erikrose @mythmon @mathjazz r?

I'll be posting this next Monday so please submit your feedback before then. Thanks!

### CODE REVIEW

**Comment by Osmose** [comment]
Doh, forgot to put the mentions in the comment instead of the description so that it sends emails.

@lonnen @peterbe @erikrose @mythmon @mathjazz r?

**Comment by peterbe** [comment]
r+

**Comment by mythmon** [comment]
r+

Also, fwiw, I got the notification from the first mention too.

**Comment by mathjazz** [comment]
r+

File: blog/extravaganza/2015-07-July.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+Once a month, web developers from across Mozilla get together to dumpster dive
+for hardware to add to our in-house cloud computing service. While we argue
+about the compute power of TI-83s, we find time to talk about the work that
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: -that

File: blog/extravaganza/2015-07-July.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+we've shipped, share the libraries we're working on, meet new folks, and talk
+about whatever else is on our minds. It's the Webdev Extravaganza! The meeting
+is open to the public; you should stop by!
+
+You can check out the [wiki page][wiki] that we use to organize the meeting, or
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: -that

File: blog/extravaganza/2015-07-July.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+a summary!
+
+## Shipping Celebration
+The shipping celebration is for anything we finished and deployed in the past
+month, whether it be a brand new site, an upgrade to an existing one, or even a
+release of a library.
+
+### DXR search package for Atom
+First up was [Osmose][] (that's me!) with a package for [Atom][], a text editor
+made by Github. The package is called [atom-dxr-search][] and allows you to
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: GitHub

File: blog/extravaganza/2015-07-July.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+perform searches on [DXR][] from directly within your text editor. And, if you
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: A parenthetical like "(Mozilla's structured code query engine)" might help newbs.

File: blog/extravaganza/2015-07-July.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+test infrastructure changes before they get deployed.
+
+[lonnen]: https://mozillians.org/u/lonnen/
+[Socorro]: http://socorro.readthedocs.org/en/latest/
+[AWS]: http://aws.amazon.com/
+[Atlas]: https://atlas.hashicorp.com/
+
+### Air Mozilla /new page
+[Peterbe][] launched the [New/Upload][] page on [Air Mozilla][], which allows
+users to upload existing videos or record a new video from their webcam within
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: "allows users to" → "lets users": less formal, shorter

File: blog/extravaganza/2015-07-July.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+the webpage for posting on the site. The page makes it much easier to submit
```
Comment (erikrose) [inline blog/extravaganza/2015-07-July.md]: Maybe we don't even need "within the webpage for posting on the site".

**Comment by erikrose** [comment]
Otherwise, r+!

**Comment by lonnen** [comment]
ship it

**Pull request merged**